Big-Spender/readme.md

```diff
@@ -112,7 +114,7 @@ INSERT YOUR QUERY HERE
 **You:** Then you need an extra clause. Here's the query:
 
 ```sql
-CREATE YOUR QUERY HERE
+SELECT su.supplier, sum(s.amount), date FROM spends s JOIN suppliers su ON s.supplier_id = su.id WHERE date IN ('2021-03-01', '2021-04-01') GROUP BY su.supplier, date ORDER BY supplier;
```

Comment:
nit: it's better to put the columns of the grouping first, then put the columns that get aggregated.

i.e. SELECT su.supplier, date, sum(s.amount)